sass/ting_extended_search.scss

```diff
@@ -20,185 +20,153 @@ $tablet: "(max-width: 1024px)";
 
 // Extended search
 // ========================================================================== //
+.pane-search-form  {
+  width: 75%;
+}
+.site-header .header-inner .pane-search-form .form-item-search-block-form {
+  width: 75%;
+}
+.pane-search-form {
+  position: relative;
+  #search-block-form {
+    .form-item-search-block-form {
+      width: 75%;
+    }
 
-.ding2-site-template {
-  .site-header {
-    .header-inner {
-      .pane-search-form .form-actions {
-        #search-extend-term {
-          right: 170px;
+    .form-actions {
+      float: left;
+      position: absolute;
+      width: auto;
+      right: 197px;
 
-          @media #{$tablet} {
-            right: 140px;
-            width: 60%;
-          }
+      > a {
+        float: left;
+        display: inline-block;
+        margin: -2px 10px 0 0;
+        line-height: 60px;
+
+        @include min-height(60px);
+
+        // Media from-medium
+        @include from-medium() {
```

Comment:
Do this changes in ting_extended_search:
1) Remove in file config.rb line 2 require 'zen-grids',
2) Remove in file ting_search_terms.scss line 1, 5, and include on first line @import "../../../themes/ddbasic/sass/_base.scss";
3) Change @include from-medium() => @include media($tablet)
    @include small-medium()/@include small()  => @include media($mobile)
4) Remove file _mixins.scss